webpack/private/webpack_devserver.bzl

```diff
@@ -46,17 +88,25 @@ def webpack_devserver(
     if chdir:
         unwind_chdir_prefix = "/".join([".."] * len(chdir.split("/"))) + "/"
 
+    webpack_configs = _create_configs(name, entry_point, entry_points, webpack_config, chdir)
```

Comment:
`chdir = kwargs.pop("chdir", None)` should be removed above since it is now a param